migrate/20250225_add_loadbalancer_ports.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+TYPE_LOAD_BALANCER_PORT = "1p"
```

Comment:
As you are referencing UBID below, you will need the `require_relative "../ubid"`.  That's the cause of the PR failing CI currently.